README.md

```diff
@@ -171,6 +171,10 @@ DevOps is the combination of cultural philosophies, practices, and tools that in
 - [Meshery](https://meshery.io/) - An open source, cloud native manager that enables the design and management of all Kubernetes-based infrastructure and applications.
 - [Digger](https://digger.dev) - Open Source Infrastructure as Code management tool that runs within your CI/CD system.
 
+## Productivity Tools
```

Comment:
Cn you also add the section on the `CONTENTS`? Ex. here https://github.com/wmariuss/awesome-devops/blob/main/README.md#contents

Reply:
I will add it.